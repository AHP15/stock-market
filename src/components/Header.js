import { Link } from 'react-router-dom';
import styles from '../styles/Header.module.css';

import left from '../assets/left.png';

const Header = () => (
  <header className={styles.header}>
    <Link to="/" className={styles.back}>
      <img src={left} alt="back" />
    </Link>
    <p>US tech companies</p>
  </header>
);
export default Header;
