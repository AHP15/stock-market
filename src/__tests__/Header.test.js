import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import { BrowserRouter } from 'react-router-dom';

import Header from '../components/Header';

it('renders Header correctly', () => {
  const tree = renderer.create(<BrowserRouter><Header /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
