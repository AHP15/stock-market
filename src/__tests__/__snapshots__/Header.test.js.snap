// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Header correctly 1`] = `
<header
  className="header"
>
  <a
    className="back"
    href="/"
    onClick={[Function]}
  >
    <img
      alt="back"
      src="left.png"
    />
  </a>
  <p>
    US tech companies
  </p>
</header>
`;
